brainrender_napari/data_models/atlas_table_model.py

```diff
@@ -55,6 +56,20 @@ def data(self, index: QModelIndex, role=Qt.DisplayRole):
         if role == Qt.ToolTipRole:
             hovered_atlas_name = self._data[index.row()][0]
             return self.view_type.get_tooltip_text(hovered_atlas_name)
+        if role == Qt.BackgroundRole:
+            atlas_name = self._data[index.row()][0]
+            local_version = self._data[index.row()][2]
+            if local_version == "n/a":
+                # Remote atlas: greyed out background
+                return QBrush(Qt.lightGray)
+            else:
+                updated = get_atlases_lastversions()[atlas_name]["updated"]
```

Comment:
The performance decrease is quite bad, as you noticed, and will be due to this line of code. This line of code will make an API call to the remote data repo every time the Qt refreshes, which is a LOT :laughing: 

We do however, already have the information we need in the table.
I suggest we fix this by comparing values in the local_version and the latest_version columns of the table.